docs/UserGuide.md

```diff
@@ -75,20 +75,20 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a contact: `add`
 
-Adds a person to the address book.
+Adds a contact to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GITHUB_LINK [t/TAG] [ts/TECH_STACK]…​`
```

Comment:
is this supposed to be github link or username? the first example add (on top) seems as though it is username